### ISSUE TITLE
PlantUML

### ISSUE THREAD

**Issue by HilkopterBob** [simple-icons/simple-icons#9629]
### Brand Name

PlantUML

### Website

https://plantuml.com/de/

### Popularity Metric

The SW-Rank is ~120k
https://www.similarweb.com/website/plantuml.com/#overview

### Official Resources for Icon and Color

None

### Additional Comments

Context:
PlantUML is a widely used companion tool with selfhosted GitLab-Instances. PlantUML is an diagram rendering engine, like mermaid. Its used for embedding plantuml-code in readme's and rendering them dynamically as image-file.

### Contributing

No

**Comment by tomap** [comment]
Indeed. I always thought the emoji on their front page was the logo, but it's just a décoration (the little green plant , 🌱)

**Comment by tekiegirl** [comment]
+1 to this request.
The colours on the roof of the plant/building are the same as the UML logo, for which you chose #FABD14.
Another reference: https://www.google.com/imgres?imgurl=https://cdn-0.plantuml.com/logo3.png&tbnid=DvXxXMQt3ILmBM&vet=10CBEQMyi0AmoXChMIyNfIo6XOgwMVAAAAAB0AAAAAEAQ..i&imgrefurl=https://plantuml.com/es-dark/sequence-diagram&docid=sueA3Tli6f3jpM&w=151&h=151&ved=0CBEQMyi0AmoXChMIyNfIo6XOgwMVAAAAAB0AAAAAEAQ

**Comment by PeterShaggyNoble** [comment]
Thanks for the confirmation, @tomap & @tekiegirl. Would either of you, or you @HilkopterBob, like to have a read through [our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) and take this on yourself?
